stable/syncthing/2.1.0/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: syncthing
+version: 2.1.0
+upstream_version: 1.15
```

Comment:
Are you using a upstream helmchart to build this?
`Upstream_version` refers to a upstream helmchart, not version of the app (thats what `app_version` refers to, which we don't currently use)